doc/release/1.15.0-notes.rst

```diff
@@ -118,7 +118,7 @@ C API changes
 =============
 
 ``NpyIter_Close`` has been added and should be called before
-``NpyIter_Dealloc`` to resolve possible writeback-enabled arrays.
+``NpyIter_Deallocate`` to resolve possible writeback-enabled arrays.
```

Comment:
Would be tempted to add a bullet point to match the other sections without headers in this document.

Reply:
added